### DISCUSSION TITLE
Why does it always rejected my application without a reason

### DISCUSSION THREAD

**Discussion by Miaoyww** [community/community#63103] [GitHub Education]
### Select Topic Area

Question

### Body

A few previous applications at least have reasons like 

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

But almost 10 applications i've been rejected without a reason, it only told me i was rejected, like this

Unfortunately, we weren't able to approve your educational discount request for [@Miaoyww](https://github.com/Miaoyww).

What happened?
Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

What i should do?I have exactly no idea

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Miaoyww , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by ianchb** [comment]
Same. I used to be verified easily in 2019 and 2022 - in one go. However, I got rejected for more than 80 times this time.
I must say that some of GitHub's new updates are making more trouble, preventing real student from getting the benefits.

**Comment by Miaoyww** [comment]
No one replies for sure?🥺
